pkg/serverconfig/serverconfig.go

```diff
@@ -0,0 +1,10 @@
+package serverconfig
+
+import "github.com/weaveworks/common/server"
+
+type ServerConfig struct {
+	server.Config `yaml:",inline"`
+	ClientCert    string `yaml:"client_cert_file"`
+	ClientKey     string `yaml:"client_key_file"`
+	ServerName    string `yaml:"server_name"`
```

Comment:
Maybe these can be defined in the integrations manager config instead since it only affects that system?